tests/wpt/metadata/html/semantics/forms/form-submission-target/rel-form-target.html.ini

```diff
@@ -1,22 +1,13 @@
 [rel-form-target.html]
-  [<form rel="noopener noreferrer"> with <form target>]
-    expected: FAIL
+ [<form rel="opener noopener"> with <form target>]
```

Comment:
It looks like the indentation in this file is incorrect as well:
`ParseError: Unexpected indent: /repo/tests/wpt/metadata/html/semantics/forms/form-submission-target/rel-form-target.html.ini line 5`

 Each new level needs to have a two-space indent.